packages/react-native-windows-init/src/Cli.ts

```diff
@@ -219,13 +217,12 @@ function getLatestMatchingVersion(
       // Ideally we'd be able to just use npm.packages.range(pkg, versionSemVer) here,
       // but alas it fails to give us the right result for react-native-windows@^0.60.0-0
       // as it fails to return pre-release versions
-      npm.packages.releases(
-        pkg,
-        (err: any, details: {[key: string]: Record<string, any>} | null) => {
-          if (err) {
-            reject(err);
-          } else if (details) {
-            const versions = Object.keys(details);
+      npmFetch.json(pkg, {registry: NPM_REGISTRY_URL}).then(
+        (response: Record<string, unknown>) => {
```

Comment:
Do typings for this have a type for the response here that we can lean on?

If not, consider changing the type from `Record<string, unknown>` to something like `{versions: Record<string, unknown>}`, since we assume that property is present later on. Makes auto-complete or catching spelling errors a bit easier.

Reply:
No, no useful types.